projects/ngx-sub-form/src/lib/ngx-sub-form.component.ts

```diff
@@ -47,6 +43,24 @@ export abstract class NgxSubFormComponent<ControlInterface, FormInterface = Cont
     }, 0);
   }
 
+  private mapControls(
+    controlTransformationCallback?: (ctrl: AbstractControl, key: string) => any,
+    checkControlCallback: (ctrl: AbstractControl) => boolean = () => true,
```

Comment:
This would probably better named `filterControl`

Reply:
Done